selftests/functional/test_loader.py

```diff
@@ -215,6 +215,9 @@ def test_load_not_a_test(self):
     def test_load_not_a_test_not_exec(self):
         self._test('notatest.py', NOT_A_TEST, 'NOT_A_TEST')
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
@ldoktor if you still disagree, you can propose a revert.  Then, if it convinces the reviewer, it can be reverted.

BTW, when you mention "it takes 0.4s", you're talking about what would happen in a majority of environments, but we've seen that it's not the case for **all** environments:

https://kojipkgs.fedoraproject.org/work/tasks/9189/19059189/build.log

This particular test was run with Avocado 47.0, in which **no fork bomb condition exists**.  Still, it failed due to timing issues.

Is that clear enough?